photutils/isophote/tests/test_angles.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import math
+
+import numpy as np
+import unittest
```

Comment:
I was wondering about that too. I can convert the unit tests to pytest if that's what we want.